tests/integration/features/bootstrap/FeatureContext.php

```diff
@@ -580,11 +580,44 @@ public function userSeesAttendeesInRoom(string $user, string $identifier, int $s
 
 		if ($formData instanceof TableNode) {
 			$attendees = $this->getDataFromResponse($this->response);
+		} else {
+			$attendees = [];
+		}
+		$this->assertAttendeeList($identifier, $formData, $attendees);
+	}
+
+	/**
+	 * @Then /^user "([^"]*)" sees the following attendees in breakout rooms for room "([^"]*)" with (\d+) \((v4)\)$/
+	 *
+	 * @param string $user
+	 * @param string $identifier
+	 * @param int $statusCode
+	 * @param string $apiVersion
+	 * @param TableNode $formData
+	 */
+	public function userSeesAttendeesInBreakoutRoomsForRoom(string $user, string $identifier, int $statusCode, string $apiVersion, TableNode $formData = null): void {
+		$this->setCurrentUser($user);
+		$this->sendRequest('GET', '/apps/spreed/api/' . $apiVersion . '/room/' . self::$identifierToToken[$identifier] . '/breakout-rooms/participants');
+		$this->assertStatusCode($this->response, $statusCode);
+
+		if ($formData instanceof TableNode) {
+			$attendees = $this->getDataFromResponse($this->response);
+		} else {
+			$attendees = [];
+		}
+		$this->assertAttendeeList($identifier, $formData, $attendees);
```

Comment:
I think that will need to apply a order before send the `$attendees` to assertion.
This test broke.

Reply:
Sorting was already there, added the roomToken now